app/views/pages/about.html.haml

```diff
@@ -0,0 +1,30 @@
+.container.root-container
+  %h1
+    ITutor - making your web development dreams come true
+  %p
+    We're a small team developing this project as a part of the Netguru Code College program.
+  %p
+    The idea behind ITutor is to connect technology mentors with newbies looking for a reliable source of tech knowledge.
+    The app is based on the belief that participating mentors can not stand keeping their knowledge unshared,
+    and that hungry newbies are on their way to becoming mentors themselves.
+  %p
+    Initial app version implies simple connection of two groups via either internal or email messages with potential
+    further connection via platform controlled meetings. The mentors have ratings and opinions.
+    Any user is able to search for a mentor with a specific skill set and capable of teaching in a specific human language.
+  %p
+    ITutor runs completely on Ruby on Rails with Bootstrap and a handful other libraries sprinkled on top.
+
+  .container
```

Comment:
Create an array with members in about controller method and iterate through that variable to print all members 
pseudo code:
```
@members.each do |member|
  .col-sm.text-center
      = image_tag("Profile.png", class: "img-circular")
      = member
end
```